services/api-db/docker-entrypoint-initdb.d/01-migrations.sql

```diff
@@ -645,6 +645,26 @@ CREATE OR REPLACE PROCEDURE
   END;
 $$
 
+CREATE OR REPLACE PROCEDURE
+  convert_env_vars_from_varchar_to_text()
+
+  BEGIN
+    DECLARE column_type varchar(50);
```

Comment:
just a nit-picking one, can you use
```suggestion
    DECLARE column_type varchar(300);
```
so that is is exactly to what you expect the column_type to be :)

Reply:
As far as I understand it, this local variable is being used to store the string which describes the type of the `value` column. That type string should fit easily within 40 chars, but there's no harm in giving it a 300 character limit, either. :)